src/hooks/useVisitor.ts

```diff
@@ -0,0 +1,6 @@
+const useVisitor = () => {
+  const hasVisitedBefore = localStorage.getItem('hasVisitedBefore');
```

Comment:
Can you tell me more about why you choose localStorage here? I'm curious about it.

Reply:
It's the way that I found to know if the user is a first time visitor. 
Once the user get's to the home page, I save a local storage key called `hasVisitedBefore`.  So I only show instructions on how to use Target to the first time visitors.